TweetBook/Contract/V1/ApiRoutes.cs

```diff
@@ -6,10 +6,14 @@ public static class ApiRoutes
 
         public const string Version = "v1";
 
-        public const string Base = $"{Root}/{Version}";
+        public const string Base = Root + "/" + Version;
         public static class Posts
         {
-            public const string GetAll = $"{Base}/posts";
+            public const string GetAll = Base +"/posts";
```

Comment:
I like the fact that you are consistent with the string formatting. So we use here the parenthesis only when we want the user to write a value there when calling the endpoint.